pymc/sampling/forward.py

```diff
@@ -442,6 +450,7 @@ def sample_posterior_predictive(
     sample_dims: Optional[list[str]] = None,
     random_seed: RandomState = None,
     progressbar: bool = True,
+    progressbar_theme: Optional[Theme] = None,
```

Comment:
Since it's created in global scope, the variable is mutable anyway, so may just define directly as the optional argument? No need for the `or` statement below when initiating the progress bar

```suggestion
    progressbar_theme: Optional[Theme] = default_theme,
```

Reply:
Done!